tools/aws_lab_setup/roles/common/handlers/main.yml

```diff
@@ -1,14 +1,14 @@
-- name: restart ssh
+- name: Ensure ssh is restarted
```

Comment:
I'd prefer we don't touch any file in the provisioner like this file because we have an epic PR in #249 to merge in before we lock down that down until we rebuild it.

Reply:
Right, but that's why we resolve conflicts. I don't mind doing that if #249 is merged first. But if we're not touching files that are modified in another PR, why even use Git ?

Reply:
Hehe, there's already another conflict in *examples/apache-simple-playbook/site.yml* causing me to rebase and resolve ;-)

Reply:
Conflict resolved. If any new arise I'll merge them too (and apply changes to new stuff), so you don't have to worry.